### ISSUE TITLE
⌨ Missing alt text

### ISSUE THREAD

**Issue by BrookeCameron** [GCTC-NTGC/gc-digital-talent#14518]
### ⌨ Accessibility and Expected Behaviour

**🛡️Fable tester reported:** 
When reviewing the various applicants, I noticed unlabeled graphics next to all applicants. I don't know what these images are referring to due to a lack of information.

### WCAG 2.1 Criteria

1.1.1 Non-text Content

### What browsers are you encountering the problem on?

Microsoft Edge

### 🙋‍♀️ Proposed Solution

add labels to the icons

**Comment by esizer** [comment]
Having a hardtime knowing what this could be, we do not have any graphics next to any candidate name that I am aware of 🤔

**Comment by esizer** [comment]
@gobyrne we discussed this one this morning and beleive it has to do with the badges on the screening and assessment page. While the icons themselves do have `aria-label`'s, it isn't super clear what their purpose is if you are new to the platform since they are sort of read out of context. i.e

 > 15. Estella Durgan "P", "VA", to assess graphic

If you know what to expect this is not a huge problem but it could be confusing

Though, we are removing this page anyway, so we could just avoid this and keep it in mind for the future? 🤔

**Comment by gobyrne** [comment]
Yeah, these are only on the screening and assessment page as far as I know. And we should be getting rid of that page in the next month. I'm closing this issue
